new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DeployUtil.java

```diff
@@ -310,4 +309,46 @@ public static ExecResult deployUsingRest(String host, String port,
     return result;
   }
 
+  /**
+   * Deploy application using REST API with curl utility.
+   * @param host name of the admin server host
+   * @param port node port of admin server
+   * @param userName admin server user name
+   * @param password admin server password
+   * @param cluster name of the cluster to deploy application
+   * @param archivePath local path of the application archive
+   * @param hostHeader name of the cluster to deploy application
+   * @return ExecResult 
+   */
+  public static ExecResult deployUsingRest(String host, String port,
+            String userName, String password, String cluster, 
+            Path archivePath, String hostHeader) {
+    final LoggingFacade logger = getLogger();
+    ExecResult result = null;
+    StringBuffer curlString = new StringBuffer("status=$(curl --noproxy '*' ");
+    curlString.append(" --user " + userName + ":" + password);
+    curlString.append(" -w %{http_code} --show-error -o /dev/null ")
+        .append("-H 'host: " + hostHeader  + "' ")
+        .append("-H X-Requested-By:MyClient ")
+        .append("-H Accept:application/json  ")
+        .append("-H Content-Type:multipart/form-data ")
+        .append("-H Prefer:respond-async ")
+        .append("-F \"model={ name: 'testwebapp', targets: [ { identity: [ clusters, '")
```

Comment:
should there be one more argument to the method - app name testwebapp ?

Reply:
Added a new parameter for the Application Name